src/dev/shared/SharedAssemblyInfo.cs

```diff
@@ -37,7 +37,7 @@
 
 // Please DO NOT commit changes on the 2 lines below unless
 // you're about to release a new version of the app on the Store.
-[assembly: AssemblyVersion("1.0.9.0")]
-[assembly: AssemblyFileVersion("1.0.9.0")]
+[assembly: AssemblyVersion("1.0.10.0")]
```

Comment:
Please revert the changes in this file :)

Reply:
done